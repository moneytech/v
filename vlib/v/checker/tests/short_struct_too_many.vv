struct Test {
	foo bool
}

fn main() {
	t := Test{true, false}
	_ = t
}
