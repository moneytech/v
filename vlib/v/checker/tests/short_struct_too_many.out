vlib/v/checker/tests/short_struct_too_many.v:6:7: error: too many fields
    4 |
    5 | fn main() {
    6 |     t := Test{true, false}
      |          ~~~~~~~~~~~~~~~~~
    7 |     _ = t
    8 | }
